### ISSUE TITLE
[Caffe2] Error running net train when running resnet50

### ISSUE THREAD

**Issue by gabriellap** [pytorch/pytorch#9133]
## Issue description
I am getting the following error when training resnet50:

E0703 14:25:24.046447 34382 prefetch_op.h:110] Prefetching error basic_string::_M_construct null not valid
E0703 14:25:24.046572 34336 prefetch_op.h:83] Prefetching failed.
E0703 14:25:24.046938 34336 net_simple.cc:68] Operator failed: input: "train_reader" output: "imonaboat/data_nhwc" output: "imonaboat/label" name: "" type: "ImageInput" arg { name: "std" f: 128 } arg { name: "scale" i: 256 } arg { name: "cudnn_exhaustive_search" i: 0 } arg { name: "crop" i: 224 } arg { name: "is_test" i: 0 } arg { name: "use_cudnn" i: 1 } arg { name: "batch_size" i: 32 } arg { name: "mirror" i: 1 } arg { name: "mean" f: 128 } device_option { device_type: 1 cuda_gpu_id: 0 }

## Code example
The error is here: workspace.RunNet(train_model.net.Proto().name)

I am following this tutorial: https://github.com/caffe2/tutorials/blob/master/Multi-GPU_Training.ipynb

- Caffe2:
